test/aria/utils/Time.js

```diff
@@ -142,10 +132,10 @@ Aria.classDefinition({
          */
         test_interpretTime : function () {
             var ariaTimeUtil = aria.utils.Date;
-            var useCases = ["22:4p:1", "99", "9p", "9h00", "9:9", "11:09   p", "9 9", "9/2", "9;2", "9-2", "9:9",
+            var useCases = ["99", "9h00", "9:9", "9 9", "9/2", "9;2", "9-2", "9:9",
```

Comment:
I now that part was already present, and also that a future cleaning is planned. But this isn't a lot to do: can you change the entries to object, with two properties, one holding the input value (former `useCases` entry) and one holding the expected result (former `checker` entry).
This would be fare more logical.

Example: 

```javascript
var useCases = [{input: "99", time: "09:09:00"}];
```